### PR TITLE
docs(aws-ecs-patterns): update documentation for the aws-ecs-patterns module

### PR DESCRIPTION
**Pull request by piradeepk**
Updated the documentation in the aws-ecs-patterns module.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts [hunk 1]
```diff
@@ -166,7 +177,9 @@ export interface LoadBalancedServiceBaseProps {
  */
 export abstract class LoadBalancedServiceBase extends cdk.Construct {
   public readonly assignPublicIp: boolean;
-
+  /**
+   * The desired number of instantiations of the task definition to keep running on the service.
```
Comment (iamhopaul123) [inline packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts]: Why only "desiredCount" and "cluster" have comments?

File: packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts [hunk 2]
```diff
@@ -67,7 +63,7 @@ export interface QueueProcessingServiceBaseProps {
   readonly environment?: { [key: string]: string };
 
   /**
-   * Secret environment variables to pass to the container
+   * The secret environment variables to pass to the container
```
Comment (iamhopaul123) [inline packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts]: Perhaps you also want a period for this.

File: packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts (hunk 1 above)
Comment (pkandasamy91) [reply packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts]: This is the first pass, the doc writer will go through the rest after these are updated.

File: packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts (hunk 2 above)
Comment (pkandasamy91) [reply packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts]: Updated

File: packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts [hunk 3]
```diff
@@ -124,23 +131,27 @@ export interface LoadBalancedServiceBaseProps {
...
    */
   readonly taskRole?: IRole;
 
   /**
-   * Override value for the container name
+   * The container name value, already specified in the task definition, to be used for your service discovery service.
+   * If the task definition that your service task specifies uses the bridge or host network mode,
+   * you must specify a containerName and containerPort combination from the task definition.
+   * If the task definition that your service task specifies uses the awsvpc network mode and a type SRV DNS record is
+   * used, you must specify either a containerName and containerPort combination or a port value, but not both.
```
[the remark this answers is not in the document]
Comment (pkandasamy91) [reply packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts]: Updated

**Pull request merged**